layouts/partials/components/image.html

```diff
@@ -0,0 +1,13 @@
+{{ $image := resources.GetMatch .image.src }} {{ $image_large := $image.Resize
```

Comment:
Why not reuse https://github.com/felleslosninger/digdir-hugo-theme/blob/main/layouts/partials/components/image.html ?

Reply:
I needed to define a different sizing, and since it was hardcoded in the digdir-hugo-theme `image.html`, I didn't know any other way to easily fix this.

Reply:
Sounds like something that should be improved in the theme. Could you create an issue on the digdir-hugo-theme repo? :)

Reply:
Of course! It's done.